src/bot/commands/user_commands.py

```diff
@@ -33,26 +34,53 @@ def run_register_command(update, context):
 
 def run_get_player_recents_command(update, context):
     chat_id = update.message.chat_id
-    telegram_handle = update.message.from_user.username
 
-    registered_user = user_services.lookup_user_by_telegram_handle(telegram_handle)
```

Comment:
these changes to commands that utilize `lookup_user_by_telegram_handle` are not converting it to lowercase. add it to the lookup service to make sure its consistent as all new users will be lowercase

Reply:
Lowercase conversion happens in `lookup_user_by_telegram_handle`, see user_services.py